src/lib/UserSetting/UserSetting.php

```diff
@@ -11,10 +11,10 @@
 use Ibexa\Contracts\Core\Repository\Values\ValueObject;
 
 /**
- * @property string $identifier
- * @property string $name
- * @property string $description
- * @property string $value
+ * @property string $identifier @deprecated use {@see UserSetting::getIdentifier()} instead.
+ * @property string $name @deprecated use {@see UserSetting::getName()} instead.
+ * @property string $description @deprecated use {@see UserSetting::getDescription()} instead.
+ * @property string $value @deprecated use {@see UserSetting::getValue()} instead.
```

Comment:
Same remark as in https://github.com/ibexa/core/pull/347#discussion_r1612901570.

Reply:
Fixed via 4e13ab56.